scripts/lib/auth.js

```diff
@@ -1,5 +1,5 @@
 'use strict';
-var assign = require('lodash/object/assign');
+var merge = require('lodash/object/merge');
```

Comment:
Essentially the same thing. Meant to cut down on package size as `assign` isn’t used, but `merge` is now used in _scripts/coins-logon-widget.js_.